share/eds/cache/cache.go

```diff
@@ -0,0 +1,49 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+	logging "github.com/ipfs/go-log/v2"
+	"go.opentelemetry.io/otel"
+)
+
+var (
+	log   = logging.Logger("share/eds/cache")
+	meter = otel.Meter("eds_store_cache")
+)
+
+var (
+	ErrCacheMiss = errors.New("accessor not found in blockstore cache")
```

Comment:
[q]
Is it used anywhere besides tests? Should we make it private then?

Reply:
Not now, but it is part of Cache interface. Errors that are returned by cache are not always ErrCacheMiss.

Reply:
👌🏻

Reply:
So do you mean it will be used in a later PR?

Reply:
Made it private, will make public once there is a handler